bindingcollectionadapter/build.gradle

```diff
@@ -18,8 +18,8 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
This will break compatibility with any apps not also setting themselves to version 8, since this is still not the default as far as I am aware I don't think it's a good idea to do this.